index.js

```diff
@@ -155,12 +174,18 @@ const inventoryCommand = (message) => {
     const userData = snapshot.val();
     const userIdentification = message.author.id;
     const userInventory = userData[userIdentification].inventory;
-
+    console.log(Object.keys(userInventory).length);
     const inventoryList = Object.values(userInventory).map(
       (item) => "•  " + item.name
     );
 
-    message.reply("your inventory: \n" + inventoryList.join("\n"));
+    message.reply(
+      "your inventory: \n" +
+        inventoryList.join("\n") +
+        "\n                                                                                                      Total cards:  " +
+        Object.keys(userInventory).length +
```

Comment:
we use this so much it might be nice to assign it to a `const` further up, something like `inventorySize`

Reply:
+1